content/v1.12/concepts/managed-resources.md

```diff
@@ -3,630 +3,745 @@ title: Managed Resources
+"/knowledge-base/guides/import-existing-resources" >}}) guide for more
+information on using the `managementPolicy` to import existing resources.
+{{< /hint >}}
+
+#### Options
+* `managementPolicy: FullControl` - **Default** - Crossplane can create, change
+  and delete the managed resource. 
+* `managementPolicy: ObserveOnly` - Crossplane only imports the details of the
+  external resource, but doesn't make any changes to the managed resource. 
+
+
+<!-- vale off -->
+### providerConfigRef
+<!-- vale on -->
+
+The `providerConfigRef` on a managed resource tells the Provider which
+[ProviderConfig]({{<ref "../concepts/providers#provider-configuration">}}) to
+use when creating the managed resource.  
+
+Use a ProviderConfig to define the authentication method to use when 
+communicating to the Provider.
+
+{{< hint "important" >}}
+If `providerConfigRef` isn't applied, Providers use the ProviderConfig named `default`.
+{{< /hint >}}
+
+For example, a managed resource references a ProviderConfig named 
+{{<hover label="pcref" line="6">}}user-keys{{</hover>}}.
+
+This matches the {{<hover label="pc" line="4">}}name{{</hover>}} of a ProviderConfig.
+
+```yaml {label="pcref"}
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: Instance
+spec:
+  forProvider:
+    # Removed for brevity
+  providerConfigRef: user-keys
 ```
 
-{{< /tab >}}
-{{< tab "GCP" >}}
+```yaml {label="pc"}
+apiVersion: aws.crossplane.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: user-keys
+# Removed for brevity
+```
+
+{{< hint "tip" >}}
+Each managed resource can reference different ProviderConfigs. This allows
+different managed resources to authenticate with different credentials to the
+same Provider. 
+{{< /hint >}}
 
-The GCP provider supports provisioning a [CloudSQL][cloudsql] instance with the
-`CloudSQLInstance` managed resource it adds to Crossplane.
+<!-- vale off -->
+### providerRef
+<!-- vale on --> 
 
-```yaml
-apiVersion: database.gcp.crossplane.io/v1beta1
-kind: CloudSQLInstance
+Crossplane deprecated the `providerRef` field in `crossplane-runtime` 
+[v0.10.0](https://github.com/crossplane/crossplane-runtime/releases/tag/v0.10.0). 
+Managed resources using `providerRef`must use [`providerConfigRef`](#providerconfigref).
+
+
+<!-- vale off -->
+### writeConnectionSecretToRef
+<!-- vale on --> 
+
+When a Provider creates a managed resource it may generate resource-specific
+details, like usernames, passwords or connection details like an IP address. 
+
+Crossplane stores these details in a Kubernetes Secret object specified by the
+`writeConnectionSecretToRef` values. 
+
+For example, when creating an AWS RDS database instance with the Crossplane 
+[community AWS
+provider](https://marketplace.upbound.io/providers/crossplane-contrib/provider-aws/v0.40.0) 
+generates an endpoint, password, port and username data. The Provider saves
+these variables in the 
+{{<hover label="secretname" line="9" >}}writeConnectionSecretToRef.name{{</hover>}}
+field. 
```

Comment:
i think it would be more accurate to say:

```
The Provider saves these variables in the secret referred to by
the `writeConectionSecretToRef.name` field.
```